webhook/config/manager/kustomization.yaml

```diff
@@ -12,4 +12,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/bpradipt/peer-pods-webhook
+  newName: quay.io/confidential-containers/peer-pods-webhook
```

Comment:
maybe better to have implicit tag? default is latest?

Reply:
Yeah I agree. The current webhook build targets don't have support to add tags like we do for the caa container. I'll add this logic in a separate PR.